runtime/src/bank.rs

```diff
@@ -170,10 +170,18 @@ use {
     },
 };
 
+struct StakeReward {
+    pubkey: Pubkey,
+    reward: RewardInfo,
+    balance: u64,
+    account: AccountSharedData,
+}
+
 #[derive(Debug, Default)]
 struct RewardsMetrics {
     load_vote_and_stake_accounts_us: AtomicU64,
     calculate_points_us: AtomicU64,
+    redeem_rewards_us: AtomicU64,
```

Comment:
does not have to be atomic.
Probably also the other ones, but those can be fixed in a separate commit.

Reply:
any reason to keep this atomic?